src/modules/site-v2/templates/primary/map.html

```diff
@@ -221,7 +225,7 @@
       <div class="popup">
         <ul>
           ${d.species ? `<li><strong>Species:</strong> <i>${d.species}</i></li>` : ''}
-          ${d.isotype ? `<li><strong>Isotype:</strong> ${d.isotype}</li>` : ''}
+          ${d.isotype ? '<li><strong>Isotype:</strong> <a href="../isotype/' + d.isotype + '">' + d.isotype + '</a></li>' : ''}
```

Comment:
instead of string concatenation ('' + ''), consider using the same string interpolation `\`backtick with ${var}\`` of the surrounding code, eg:
```js
          ${d.isotype ? `<li><strong>Isotype:</strong> <a href="../isotype/${d.isotype}">${d.isotype}</a></li>` : ''}

```